bot/interact.go

```diff
@@ -81,6 +81,10 @@ func interactAPIHandler(botContext cmd.BotContext) http.HandlerFunc {
 					if minSalary > maxSalary {
 						validationErrors["min_salary"] = "The Salary Min field should contain a lower value than the specified in Salary Max field."
 					}
+
+					if (minSalary * 5) > (maxSalary * 2) {
```

Comment:
this should be more understandable
```suggestion
					if 2.5*float32(minSalary) > float32(maxSalary)  {
```

Reply:
Whichever is more idiomatic. I'm fine with either option.